EXAMPLES

```diff
@@ -207,6 +212,7 @@ Documentation using sphinx_rtd_theme
 * Free your information from their silos (French):
   http://redaction-technique.org/ (customized)
 * Releases Sphinx extension: https://releases.readthedocs.io/
+* RPM Packaging Guide: https://rpm-packaging-guide.github.io/
```

Comment:
This has moved to asciidoc.

>The document itself was originally written in sphinx-doc reStructuredText and was published at http://rpm-guide.readthedocs.io/en/latest/.
>
>The document is now being converted to AsciiDoc and its draft is published at https://rpm-packaging-guide.github.io/. Note that this is a work in progress.

https://github.com/redhat-developer/rpm-packaging-guide